src/visitor.js

```diff
@@ -144,6 +144,10 @@ class VisitState {
             path.node.body.unshift(T.expressionStatement(increment));
         } else if (path.isStatement()) {
             path.insertBefore(T.expressionStatement(increment));
+        } else if ((path.isFunctionExpression() || path.isArrowFunctionExpression()) && T.isVariableDeclarator(path.parentPath)) {
+            path.parentPath.parentPath.insertBefore(T.expressionStatement(
+                increment
```

Comment:
Given there are 3 instances of `T.expressionStatement(increment)` in this function block now, would it be good to make this a variable now?